data/discord_messages.yaml

```diff
@@ -1423,3 +1418,21 @@ bp:
 
 
     If that's not working out, this is a good secondary spot for help.
+python:
+  description: "Python"
+  content: >-
+    Python 3.9 is the minimum version of Python supported by Home Assistant,
+    with 3.9 being recommended. If you're using a Python virtual environment
+    install you can install an updated version of Python, then manually upgrade
+    your venv as described in
+    <https://blog.ceard.tech/2017/12/upgrading-python-virtual-environment.html>
+check:
+  description: "Check configuration"
```

Comment:
What about the button in the UI? 🙃

Reply:
The reason this command is used is because the UI triggered check does not catch things like duplicate keys

Reply:
That's weird 🙃 wonder if that could be solved.